src/api/teams/team_repos.rs

```diff
@@ -0,0 +1,114 @@
+use crate::params;
+use crate::{Octocrab, Result};
+use reqwest::StatusCode;
+
+/// Handler for managing a team's repositories through
+/// GitHub's teams API.
+///
+/// Created with [`TeamHandler::repos`]
+///
+/// [`TeamHandler::repos`]: ./struct.TeamHandler.html#method.repos
+pub struct TeamRepoHandler<'octo> {
+    crab: &'octo Octocrab,
+    org: String,
+    team: String,
+}
+
+/// Whether a team manages a repository.
+pub enum ManagesRepo {
+    Yes,
+    No,
+    Err,
+}
+
+impl<'octo> TeamRepoHandler<'octo> {
+    pub(crate) fn new(crab: &'octo Octocrab, org: String, team: String) -> Self {
+        Self { crab, org, team }
+    }
+
+    /// Checks if a team manages a repository.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let manages_repo = octocrab::instance()
+    ///     .teams("owner")
+    ///     .repos("team")
+    ///     .manages("owner", "repo")
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn manages(
+        &self,
+        repo_owner: impl Into<String>,
+        repo_name: impl Into<String>,
+    ) -> Result<ManagesRepo> {
+        let url = format!(
+            "/orgs/{org}/teams/{team}/repos/{owner}/{repo}",
+            org = self.org,
+            team = self.team,
+            owner = repo_owner.into(),
+            repo = repo_name.into(),
+        );
+        let res = self.crab._get(&url, None::<&()>).await?;
+        Ok(match res.status() {
```

Comment:
I would remove the `ManagesRepo` type, and instead return the alternative JSON response that contains the repository with permissions https://developer.github.com/v3/teams/#check-if-a-team-manages-a-repository Then people will be able to check what permissions the team has and in the simple case you can write
```rust
if octocrab::instance().teams("owner").repos("team-name").check_manages("owner", "repo").is_ok() {
  // ...
}